source/jormungandr/jormungandr/scenarios/journey_filter.py

```diff
@@ -85,24 +94,33 @@ def filter_journeys(responses, instance, request):
     composed_filter = ComposedFilter()
     [composed_filter.add_filter(f) for f in filters]
 
-    return composed_filter.compose_filters()(get_qualified_journeys(responses))
+    if is_debug:
```

Comment:
Is there a test that allows to see the debug mode works ?

Reply:
I just added a test on the wrapper: https://github.com/CanalTP/navitia/pull/2438/files#diff-ef2db348430687b00e4f116823df8d08R887
Not sure that's what you had in mind, hope so :stuck_out_tongue: